elegant-business/style-jetpack.css

```diff
@@ -1,3 +1,5 @@
+/*THIS FILE IS COMPILED FROM AN .SCSS FILE - DO NOT EDIT DIRECTLY*/
+/*THIS FILE IS COMPILED FROM AN .SCSS FILE - DO NOT EDIT DIRECTLY*/
```

Comment:
Delete one

Reply:
For strange things like this (where those double-lines exist in SVN but not in GIT) 'deleting one' will mean changing that file in GIT and SVN since this change represents how it exists in SVN today.

For the purpose of simplicity I would prefer to avoid as many 'double repository changes' as we can and instead fix them as we progress in the future once parity has been reached.  If there's no harm in bringing into GIT what exists in SVN doing so is my preference.